fprodents/workflows/bold/base.py

```diff
@@ -242,6 +242,7 @@ def init_func_preproc_wf(bold_file):
     outputnode = pe.Node(
         niu.IdentityInterface(
             fields=[
+                "bold_mask",
```

Comment:
it seems it needs the input

Reply:
actually, i think this is
```suggestion
                "anat_mask",
```

Reply:
Specifically, we output the mask calculated from the structural in bold space. Do you think it's better to call it `bold_mask`, `anat_mask_bold`, or something else entirely?

Reply:
sorry, I did not realize this is the outputnode, scratch my comment